src/courseware/course/sidebar/sidebars/course-outline/CourseOutlineIcon.jsx

```diff
@@ -0,0 +1,35 @@
+import { injectIntl, intlShape } from '@edx/frontend-platform/i18n';
+import PropTypes from 'prop-types';
+
+import messages from './messages';
+import { ReactComponent as OutlineIcon } from './outline-icon.svg';
+
+const CourseOutlineIcon = ({
+  intl,
+  status,
+}) => (
+  <>
+    <span className="pgn__icon m-0 m-auto" aria-label={intl.formatMessage(messages.openCourseOutlineTrigger)}>
+      <OutlineIcon />
+    </span>
+    {status === 'active'
+      ? (
+        <span
+          className="rounded-circle p-1 position-absolute"
+          data-testid="course-outline-dot"
+          style={{
+            top: '0.3rem',
+            right: '0.55rem',
+          }}
```

Comment:
[question]: Can we transfer inline styles to a SCSS file?

Reply:
This file will be removed in the next PR: https://github.com/raccoongang/frontend-app-learning/pull/17